app/controllers/child_item_requests_controller.rb

```diff
@@ -0,0 +1,87 @@
+class ChildItemRequestsController < ApplicationController
+  before_action :authenticate_user!
+
+  def toggle_picked_up
+    child_item_request.toggle!(:picked_up)
```

Comment:
Given the change this necessitated to the rubocop config, Im curious what the advantage of this is over `update(picked_up: !picked_up`? we could wrap this in a method on the `ChildItemRequest` model like...

```
def pick_up!
  update!(picked_up: true)
end

def undo_pick_up!
  update!(picked_up: false)
end
```

This maybe _feels_ a bit more semantic to me, but I dont feel super strongly about it.

Reply:
I'm  happy to update it but it does feel like we're just doing it to make rubocop happy, not because we have an advantage from a code perspective. If later we add validation on the model, we would have to specifically ignore the picked_up attribute... Not a big. Let me know if this changes your mind, otherwise I'll incorporate the snippet above.